samples/kotlin-android-app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
```

Comment:
Question: Do we need this permission for all Android 6.0 (Marshmallow) and higher like what we mentioned [here](https://www.docs.developers.amplitude.com/data/sdks/android/#add-dependencies)?  Or this is related to the migration logic?

Reply:
Was added in initial commit 'feat: add remnant events migration plugin Marvin Liu 6/24/22, 11:27 AM'
Looks like it can be removed - I run the sample application without it (Android emulator).
Removed.